common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -781,5 +784,17 @@ public static ConscryptHostnameVerifier getHostnameVerifier(TrustManager trustMa
         return toConscrypt(trustManager).getHostnameVerifier();
     }
 
-
+    /**
+     * Wraps the HttpsURLConnection.HostnameVerifier into a ConscryptHostnameVerifier
+     */
+    public static ConscryptHostnameVerifier wrapHostnameVerifier(final HostnameVerifier verifier) {
+        // needed to add final due to : error: local variable verifier is accessed from within inner class; needs to be declared final
```

Comment:
Ah yes, I forgot to mention, you should save the value in a field then it doesn't need to be final.